.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "mayastor"]
+	path = mayastor
+	url = git@github.com:openebs/mayastor-extensions.git
+	branch = expose_plugin
```

Comment:
note: remember to fix branch here prior to merging

Reply:
@tiagolobocastro , Shall we merge this as is? I am going to keep the extension PR open for some more time. I can merge that during stem implementation PR and point it to develop.

Reply:
ok, should be fine for now